easybuild/easyconfigs/p/pkg-config/pkg-config-0.29-intel-2015b.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'pkg-config'
+version = '0.29'
+
+homepage = 'http://www.freedesktop.org/wiki/Software/pkg-config/'
+description = """pkg-config is a helper tool used when compiling applications and libraries. It helps you insert the
+ correct compiler options on the command line so an application can use
+  gcc -o test test.c `pkg-config --libs --cflags glib-2.0`
+ for instance, rather than hard-coding values on where to find glib (or other libraries)."""
+
+# don't use PAX, it might break.
+tar_config_opts = True
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://pkgconfig.freedesktop.org/releases/']
+
+configopts = " --with-internal-glib"
+
+sanity_check_paths = {
+    'files': ['bin/pkg-config'],
+    'dirs': []
```

Comment:
add `,` at the end (the existing pkg-config easyconfigs have it too, let's keep them consistent)